cmd/lotus-bench/bench-sectors.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
```

Comment:
I don't know the team history or view or all the context here, but in general would vote for not seeing this amount of bash getting checked in.

I generally advocate for a "two finger rule" with checked-in bash (i.e., that any bash that extends what I can cover with two fingers on the screen is a signal that instead convert to something else).  This is stemming from a code review and maintenance regard given (my opinion) shell scripting's esoteric and less-ergonomic nature.  

Basically if this is something we care about continuing to work and be modifiable in future, then I would convert it to something else (presumably go) that the team is well-versed in.

Reply:
This is here for the reverse reason! I find this more ergonomic for doing a full seal workflow than what's available in lotus-bench today. I suppose it could be replaced with a `full` but I'm not sure who wants this. I mostly checked this in because I found it useful and didn't want to either discard it or put it in a place where it could get lost; but perhaps it's not useful or interesting to anyone else. I don't know!

Reply:
Got it.  I like the idea of checking it in so we don't lose it.  It seems like we don't know the full utility of this in the long term so can leave it as is.  If this is something we need to use/maintain/support in future, we could make it more robust.  Seems good to me.

Reply:
I'll add some additional notes to this—it may turn out to not be useful and if it's not touched in 6 months then maybe it gets thrown out, that's OK.

Reply:
I've added some docs to the top of the script. We'll see what @aarshkshah1992 thinks of it being in here. I'm fine removing it entirely if you both think it's cruft.

Reply:
I had a look at this. Okay to keep it here if it's just to help out people who might want to use this as a reference to understand the full PoRep/NI-PoRep workflow. The script is straight forward to follow.

However, strongly recommend that we translate this to Golang if we're actually using this for benchmarking that workflow.